tasks/check-settings.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Check for innodb_log_file_size setting (Ubuntu)"
+  shell: 
+    cmd: grep -E ^innodb_log_file_size /etc/mysql/my.cnf | awk -F= '{ print $2}' | sed 's/\s//g'
+    removes: "/etc/mysql/my.cnf"
+  register: configured_innodb_log_file_size
+
+- name: "Abort when innodb_log_file_size changes"
+  fail:
+    msg: "The existing MySQL servers has innodb_log_file_size={{ configured_innodb_log_file_size.stdout }}, but your are trying to set mysql_innodb_log_file_size={{ mysql_innodb_log_file_size }}. Please, change this value in server or the variable value in ansible. See: https://dev.mysql.com/doc/refman/5.6/en/innodb-redo-log.html"
```

Comment:
Please change the error message to something that reads better, such as:
 msg: "The existing MySQL server has innodb_log_file_size={{ configured_innodb_log_file_size.stdout }}, but your are trying to set it to {{ mysql_innodb_log_file_size }}. Please, change this value for the variable in either ansible or the server itself. See: https://dev.mysql.com/doc/refman/5.6/en/innodb-redo-log.html"

Reply:
Thanks! Changed!